src/mp_ui_alerts.cpp

```diff
@@ -51,7 +51,37 @@ bool notif_pref(const std::string& id)
 } // end anonymous namespace
 
 // Note: This list must agree with data/gui/.../lobby_sound_options.cfg
-const std::vector<std::string> items {"player_joins", "player_leaves", "private_message", "friend_message", "public_message", "server_message", "ready_for_start", "game_has_begun", "turn_changed"};
+const std::vector<std::string> items {"player_joins", "player_leaves", "private_message", "friend_message", "public_message", "server_message", "ready_for_start", "game_has_begun", "turn_changed", "game_created"};
```

Comment:
Oh yeah, I guess you also need to address this comment here.

Reply:
I was wondering about that, but that's not in my user data folder or in the wesnoth git folder. It worked without addressing that.

Reply:
Hmm, maybe ask @Vultraz whether the comment is still relevant then? It could be that the path has changed.